src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/AnalysisEntityMapAbstractDomain.cs

```diff
@@ -70,18 +70,18 @@ protected virtual void AssertValidAnalysisData(DictionaryAnalysisData<AnalysisEn
 #endif
         }
 
-        public override DictionaryAnalysisData<AnalysisEntity, TValue> Merge(DictionaryAnalysisData<AnalysisEntity, TValue> map1, DictionaryAnalysisData<AnalysisEntity, TValue> map2)
+        public override DictionaryAnalysisData<AnalysisEntity, TValue> Merge(DictionaryAnalysisData<AnalysisEntity, TValue> value1, DictionaryAnalysisData<AnalysisEntity, TValue> value2)
```

Comment:
Same here - would suggest reverting this method and just adding a source suppression.

Reply:
💭 `value1` seems just as good as `map1` to me for this method